packages/untp-test-suite/src/config/credentials.json

```diff
@@ -1,24 +1,32 @@
-{
-  "credentials": [
-    {
-      "type": "productPassport",
-      "version": "v1.1",
-      "dataPath": "./schemas/v1.1/schema.json"
-    },
-    {
-      "type": "conformityCredential",
-      "version": "v1.0",
-      "dataPath": "./schemas/v1.0/schema.json"
-    },
-    {
-      "type": "objectEvent",
-      "version": "v2.1",
-      "dataPath": "./schemas/v2.1/schema.json"
-    },
-    {
-      "type": "transactionEvent",
-      "version": "v2.1",
-      "dataPath": "./schemas/v2.1/schema.json"
-    }
-  ]
-}
+[
+  {
+    "type": "aggregationEvent",
+    "version": "v0.0.1",
+    "dataPath": ""
+  },
+  {
+    "type": "conformityCredential",
+    "version": "v0.0.1",
+    "dataPath": ""
+  },
+  {
+    "type": "objectEvent",
+    "version": "v0.0.1",
+    "dataPath": ""
+  },
+  {
+    "type": "productPassport",
+    "version": "v0.0.1",
+    "dataPath": ""
+  },
+  {
+    "type": "transactionEvent",
+    "version": "v0.0.1",
+    "dataPath": ""
+  },
+  {
+    "type": "transformationEvent",
+    "version": "v0.0.1",
+    "dataPath": ""
+  }
+]
```

Comment:
I think we should write a comment to let the user know where the dataPath is being read from. It could start from the root directory or from this configuration file (using either an absolute or relative path).